### DISCUSSION TITLE
Control a ThingM Blink(1) USB LED Via Custom API

### DISCUSSION THREAD

**Discussion by jescodev** [isaacrlevin/presencelight#822] [Q&A]
I wanted to control the ThingM Blink(1) USB LED via the custom API interface of the PresenceLight Application.
The Blink(1) USB LED can be controlled via command line command: 

curl http://localhost:8934/blink1/fadeToRGB?rgb=%23ff0000

The Blink(1) API on PC is via the Blink1Control2 application, with the API Server Configuration option set to Start API Server, port 8934, and host set to localhost.

I tried using the PresenceLight Configure Custom API settings:
Connect to Custom API = checked.
Then set Available: to POST with the string "http://localhost:8934/blink1/fadeToRGB?rgb=%23ff0000" - same as what works at command line with curl.

I tried some other POST strings - with and without the curl command, but am not able to have any control of the USB LED via PresenceLight Custom API. 
If there are any examples of working Custom API strings for a POST command, that may help or if I am missing something please advise. Thank you!

**Comment by isaacrlevin** [comment]
Please open an issue
